src/logic/calculate.js

```diff
@@ -0,0 +1,20 @@
+import operate from './operate';
+
+const calculate = (obj, btnName) => {
+  if (btnName === '+/-') {
```

Comment:
- [ ] Kindly consider destructuring your `obj` to avoid repetition of reference, this can be done like so:
`const {total, next, operation} = obj `